src/com/projectkorra/rpg/RPGMethods.java

```diff
@@ -42,14 +44,14 @@ public static boolean checkEveryInterval(World world) {
 	}
 	
 	public static void cycleAvatar(BendingPlayer bPlayer) {
-		if (Bukkit.getOnlinePlayers().size() <= 1) return; //Don't bother with 1 person on...
```

Comment:
This comment was correct seeing as though the check was less than or equal to one.